client/blocks/comments/comment-approve-action.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { noop } from 'lodash';
+// import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import classnames from 'classnames';
+
+
+const CommentApproveAction = ( { translate, status, onClick } ) => {
+	const isApproved = status === 'approved';
+	const buttonStyle = classnames( 'comments__comment-actions-approve', {
+		'is-approved': isApproved
+	} );
+
+	return (
+		<button className={ buttonStyle } onClick={ onClick }>
+			<Gridicon icon="checkmark" size={ 18 }/>
+			<span className="comments__comment-actions-like-label">{ isApproved ? translate( 'Approved' ) : translate( 'Approve' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Approved', { context: 'comment status'} )` *ES Score: 20*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/comment-moderation-tools-on-post)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).